sequencer/src/options.rs

```diff
@@ -104,6 +104,32 @@ pub struct Options {
     #[derivative(Debug(format_with = "Display::fmt"))]
     pub state_relay_server_url: Url,
 
+    /// URL of the Auction Results Solver
+    #[clap(
+        long,
+        env = "ESPRESSO_AUCTION_RESULTS_SOLVER_URL",
+        default_value = "http://localhost:8083"
+    )]
+    #[derivative(Debug(format_with = "Display::fmt"))]
+    pub auction_results_solver_url: Url,
+
+    /// URL of generic builder
+    #[clap(
+        long,
+        env = "ESPRESSO_GENERIC_BUILDER_URL",
+        default_value = "http://localhost:8083"
```

Comment:
Same as solver, can you please make it 31004 to conform to .env?

Reply:
fixed c5bb9849f86685a5a58e8c6d48e3190b4fc9e564